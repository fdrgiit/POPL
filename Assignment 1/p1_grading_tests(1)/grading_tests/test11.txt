
<=<<>><INT>12.345.00

## INT

122222.3333INT		A000000000.00000INT		 & & % $	REALSTRING

	 A0000000001.0REAL900

80880Y.1INTINT

