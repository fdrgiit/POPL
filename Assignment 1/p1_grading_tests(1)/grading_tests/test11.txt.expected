2 29 <=
2 31 <<
2 32 >>
2 28 <
2 10 INT
2 27 >
2 34 12 
2 33 .
2 34 345 
2 33 .
2 34 0 
2 34 0 
4 38 ERROR
4 38 ERROR
4 10 INT
6 34 122222 
6 33 .
6 34 3333 
6 10 INT
6 37 A000000000 
6 33 .
6 34 0 
6 34 0 
6 34 0 
6 34 0 
6 34 0 
6 10 INT
6 38 ERROR
6 38 ERROR
6 38 ERROR
6 38 ERROR
6 37 REALSTRING 
8 36 A0000000001.0REAL 
8 34 900 
10 34 80880 
10 35 Y.1INT 
10 10 INT
